dev.linwood.butterfly.json

```diff
@@ -0,0 +1,93 @@
+{
+    "app-id": "dev.linwood.butterfly",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "butterfly",
+    "separate-locales": false,
+    "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--socket=pulseaudio",
```

Comment:
```suggestion
```
I'd recommend removing the pulseaudio permission as the app doesn't use any audio